src/kern/npf_tableset.c

```diff
@@ -456,13 +456,18 @@ npf_table_getid(npf_table_t *t)
  * npf_table_check: validate the name, ID and type.
  */
 int
-npf_table_check(npf_tableset_t *ts, const char *name, uint64_t tid, uint64_t type)
+npf_table_check(npf_tableset_t *ts, const char *name, uint64_t tid, uint64_t type, bool replacing)
```

Comment:
Please wrap long lines as follows:
```c
npf_table_check(npf_tableset_t *ts, const char *name, uint64_t tid,
    uint64_t type, bool replacing)
```
4-space indentation for the continuation (from the point of the previous line indentation).